scaife_viewer/views.py

```diff
@@ -302,13 +303,16 @@ def search_json(request):
 
 
 def morpheus(request):
-    if "word" not in request.GET:
-        raise Http404()
+    if ("word" not in request.GET) or ("lang" not in request.GET):
+        raise SuspiciousOperation()
```

Comment:
I am glad you caught your error and did so moments before I commented on it :-)

However, I would not recommend the use of `SuspiciousOperation` here. I feel this is overloading the meaning where it is mostly used by Django for security purposes. I suggest you use `HttpResponseBadRequest`.